todo-mvc-kitchensink/README.md

```diff
@@ -0,0 +1,78 @@
+# Dojo2 TodoMVC Kitchen Sink Example
+
+A feature-rich version of TodoMVC, built in current pre-release versions of Dojo 2 packages.
+
+- [Pre-requisites](#pre-requisites)
+- [Usage](#usage)
+- [Features](#features)
+- [How do I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Pre-requisites
+
+You need to be running node `v6+` and npm `v3+`.
```

Comment:
v6 and v3 may confuse people, let's be more explicit please. And I thought we supported Node 4?